frontend/templates/pages/clients/dashboard/_rows.html

```diff
@@ -11,7 +12,10 @@
                     </button>
                 </div>
                 <div class="tooltip" data-tip="Delete">
-                    <button class="btn btn-outline btn-error btn-sm btn-disabled">
+                    <button class="btn btn-outline btn-error btn-sm"
+                            hx-delete="{% url 'api:clients:delete' id=client.id %}"
+                            hx-target="#items"
```

Comment:
Does this not delete the entire list?

Reply:
No, only the desired client is gone from table.

Reply:
before:
![Screenshot from 2024-06-10 12-34-26](https://github.com/TreyWW/MyFinances/assets/119700507/7cc57971-3230-42a0-a48e-f1aa45a1c01d)

after:
![Screenshot from 2024-06-10 12-34-53](https://github.com/TreyWW/MyFinances/assets/119700507/00e835d0-a8a9-4a92-941b-1fb27c352597)